.github/workflows/ci-for-default-ns.yaml

```
@@ -75,15 +75,15 @@ jobs:
           # for testing start-debug and stop-debug deployment
           # mons
           kubectl rook_ceph debug start rook-ceph-mon-a
-          sleep 5
+          tests/github-action-helper.sh wait_for_deployment_to_be_running rook-ceph-mon-a-debug rook-ceph
```

Did you try this? Seems like it should work to call a function in that plugin script.
```suggestion
          kubectl-rook-ceph.sh wait_for_deployment_to_be_running rook-ceph-mon-a-debug rook-ceph
```

---

Yaa, I tried using it,
But it won't work as in because of it's structure, As when we called ` kubectl-rook-ceph.sh ` it will look for the following toolbox cmds let say, but won't directly call any function, 
For ex: `kubectl-rook-ceph.sh debug start` and won't like directly calling a function ` kubectl-rook-ceph.sh wait_for_deployment_to_be_running `,
But the `tests/github-action-helper.sh` entry point has this check to call a certain function.

---

Ok thanks for confirming